### PR TITLE
Add selectable rows.

### PR DESCRIPTION
**Pull request by shenst1**
@joshreeves

### CODE REVIEW

File: vendor/assets/javascripts/initializers/selectable-rows.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$(document).on('cmsify:load', function() {
+  $('.js-selectable-rows').each(function(index, el) {
```
Comment (joshreeves) [inline vendor/assets/javascripts/initializers/selectable-rows.js]: Make a `Cmsify.SelectableRows` class and then initialize it in the initializer.

File: vendor/assets/javascripts/initializers/selectable-rows.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    $el = $(el);
+    $el.find('tr').each(function() {
+      $this = $(this);
+      $this.on('click', function(e) {
+        e.preventDefault();
+        $(this).find('input').trigger('click');
```
Comment (joshreeves) [inline vendor/assets/javascripts/initializers/selectable-rows.js]: `input[type=checkbox]`

**Review by joshreeves — CHANGES_REQUESTED** [review]

File: vendor/assets/javascripts/initializers/selectable-rows.js (hunk 2 above)
Comment (shenst1) [reply vendor/assets/javascripts/initializers/selectable-rows.js]: its not necessarily a checkbox. They are also radio buttons

File: vendor/assets/javascripts/behaviors/selectable_rows.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Cmsify.SelectableRows = function(el) {
+  this.$el = $(el);
+  this.$el.find('tr').each(function() {
+    $this = $(this);
+    $this.on('click', function(e) {
+      e.preventDefault();
+      $(this).find(':radio', ':checkbox').trigger('click');
+    });
+    $this.find(':radio', ':checkbox').on('click', function(e) {
```
Comment (joshreeves) [inline vendor/assets/javascripts/behaviors/selectable_rows.js]: Can you cache the inputs like `var $inputs = $this.find(':radio', ':checkbox')`

**Review by joshreeves — COMMENTED** [review]
One small change and :ship:

**Pull request merged**